iib/workers/tasks/utils.py

```diff
@@ -175,28 +176,49 @@ def _requires_max_ocp_version(bundle):
     return False
 
 
-def serve_index_registry(db_path):
+def serve_registry(base_dir, from_index=None, fbc_dir=None, db_path=None):
```

Comment:
if you change this.. you will have to change it in build_create_empty_index.py, build_merge_index_image.py

Reply:
it would be better to do the refactoring out of this PR

Reply:
Those two does not use `serve_registry` directly.